### ISSUE TITLE
SQL: rpmbuild fails

### ISSUE THREAD

**Issue by monetdb-team** [MonetDB/MonetDB#1115]
Date: 2006-11-05 01:25:06 +0100
From: Stefan Manegold &lt;<Stefan.Manegold>&gt;
To:   Niels Nes &lt;<niels>&gt;
Version: -- development

Last updated: 2006-11-06 08:09:58 +0100

## Comment 6251
Date: 2006-11-05 13:25:06 +0100
From: Stefan Manegold &lt;<Stefan.Manegold>&gt;

Since this checkin
 ===================================================================
 2006/10/20 - nielsnes: src/include/Makefile.ag,1.11
 src/server/sql_mvc.mx,1.161
         src/storage/Makefile.ag,1.7
 src/storage/sql_storage.h,1.27
         src/storage/bat/bat_logger.mx,1.41
 src/storage/bat/bat_sequence.mx,1.13
         src/storage/bat/bat_store.mx,1.106
        
 src/test/BugDay_2005-10-06_2.9.3/Tests/bad_sql_logdir.SF-927371,1.3
        
 src/test/BugDay_2005-10-06_2.9.3/Tests/bad_sql_logdir.SF-927371.bat,1.3
        
 src/test/BugDay_2005-10-06_2.9.3/Tests/bad_sql_logdir.SF-927371.stable.err,1.3
        
 src/test/BugDay_2005-10-06_2.9.3/Tests/bad_sql_logdir.SF-927371.stable.out,1.8
        
 src/test/BugDay_2005-12-19_2.9.3/Tests/select_from_env.SF-1240701.1242164.stable.out,1.4
 more cleanup of the logger.
 Logger is now a global variable. It should now be
 possible to move it into
 a M4/5 module and reuse it in Xquery.
 one big change is the name of the GDKenv variable
 sql_logdir into logdir.
 ===================================================================
 
 rpmbuild for SQL fails with
 ========
 Processing files: MonetDB-SQL-devel-2.13.1-1_Fedora4.oid64
 error: File not found by glob:
 /var/tmp/MonetDB-SQL-root/usr/include/sql/include/*.h
 error: File not found by glob:
 /var/tmp/MonetDB-SQL-root/usr/include/sql/storage/*.h
 
 
 RPM build errors:
     File not found by glob:
 /var/tmp/MonetDB-SQL-root/usr/include/sql/include/*.h
     File not found by glob:
 /var/tmp/MonetDB-SQL-root/usr/include/sql/storage/*.h
 make: *** [rpm] Error 1
 ========
 
 basically, because the respective header files are no
 longer installed after above checkin.
 
 BUG? => fix Makefiles to in the header files again.
 FEATURE? => fix MonetDB-SQL.spec.in to not require the
 header files any more.
 

## Comment 6252
Date: 2006-11-06 08:09:58 +0100
From: Stefan Manegold &lt;<Stefan.Manegold>&gt;

Logged In: YES 
 user_id=572415
 
 fixed by Niels.
 Thanks!
 

## Comment 6253
Date: 2010-05-04 09:32:09 +0200
From: Pseudo user for Sourceforge import &lt;<sfimport>&gt;

This bug was previously known as tracker item 1590786 at http://sourceforge.net/support/tracker.php?aid=1590786
